ext/ddtrace.stub.php

```diff
@@ -23,6 +23,23 @@
      */
     const DBM_PROPAGATION_FULL = UNKNOWN;
 
+    class SpanEvent implements \JsonSerializable {
+        /**
+         * @var string The event name
+         */
+        public string $name;
+
+        /**
+         * @var int The event start time in nanoseconds, if not provided set the current Unix timestamp
+         */
+        public int $timeUnixNano;
```

Comment:
I dislike the name, but don't have a good suggestion right now.

Reply:
Sameee!!! I keep having to look at the changes to remember it, I've only been following the RFC so used that name to keep it consistent, I am open to any recommendations